portfolio/src/main/webapp/index.html

```diff
@@ -1,20 +1,20 @@
 <!DOCTYPE html>
 <html>
   <head>
-    <meta charset="UTF-8">
+    <meta charset='UTF-8'>
```

Comment:
from what I've seen, normally " is used in HTML instead of '. However its fine if you prefer ' as well

Reply:
Ok got it!